src/components/Confirm/CaptureViewer/style.css

```diff
@@ -39,6 +39,12 @@
   overflow: overlay;  /* append scrollbars if necessary and overlay over/above content (only for Chrome, to avoid double scrollbars) */
 }
 
+@media (--small-viewport) {
+  .imageWrapper {
```

Comment:
This class is also used by the standard document capture. Wouldn't this change affect it?